src/components/MainPage/Importers/Importer.vue

```diff
@@ -76,34 +68,23 @@ export default {
   data() {
     return {
       importing: false,
+      // TODO: show browser should be in config. Also maybe one for all accounts
```

Comment:
Yes and yes. In general I think the common usecase is scrape all importers and then this type of configuration is global